packages/web/src/services/solana/solana.ts

```diff
@@ -37,6 +37,27 @@ export const isSolWallet = async (destinationWallet: SolanaWalletAddress) => {
   }
 }
 
+/**
+ * Checks if the given account address is an associated token account
+ */
+export const isTokenAccount = async ({
+  accountAddress,
+  mint
+}: {
+  accountAddress: SolanaWalletAddress
+  mint: MintName
+}) => {
+  try {
+    await getTokenAccountInfo({
```

Comment:
just confirming this throws an error? I thought it could potentially return `null`.

Reply:
You are correct - I was looking at solana/web3.js not libs by accident!